### ISSUE TITLE
android not implemented.

### ISSUE THREAD

**Issue by pstanton** [aaronvb/react-native-redux-example#1]
the index.android.js is not implemented.

this is a bit misleading for beginners

**Comment by aaronvb** [comment]
@pstanton you are correct. I will work on updating the README to point that out. Thanks for the feedback!
